src/constants/routesPath.ts

```diff
@@ -1,3 +1,4 @@
 export const ROUTE = {
   HOME: "/",
+  CREATE_AUTHOR: "create-author",
```

Comment:
E đặt object để có thể thêm LIST, EDIT,...

```suggestion
  AUTHOR: {
     ADD: 'authors/add'
  }
```